package.json

```diff
@@ -147,6 +148,7 @@
     "globalthis": "^1.0.2",
     "hammerjs": "^2.0.8",
     "hash-wasm": "^4.4.1",
+    "leaflet": "^1.4.0",
```

Comment:
Does this need to be added explicitly? Won't it be added implicitly by adding `@asymmetrik/ngx-leaflet`?